src/odbc_connection.cpp

```diff
@@ -23,6 +23,9 @@
 
 #define MAX_UTF8_BYTES 4
 
+#define MB_SIZE 1048576
+#define GB_SIZE 1073741824
```

Comment:
The idea was to use these values for the "transfer buffer" if you will, so a 2GB field would still work, it would just be transfered in two `SQLGetData` calls. Not that I'm sure why you would care what the transfer buffer size would be in cases where you are transferring fields that are even bigger than that, but that was the original idea.

But now looking at the code, it appears that I never actually use `query_options.max_long_data_buffer_size`, so that will have to be addressed.